bitnami/rabbitmq/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: rabbitmq
 sources:
   - https://github.com/bitnami/bitnami-docker-rabbitmq
   - https://www.rabbitmq.com
-version: 8.16.3
+version: 8.16.4
```

Comment:
I am sorry, since you are adding a new parameter this should be a minor bump, not a patch

Reply:
Done